content-src/components/DebugPage/DebugPage.js

```diff
@@ -28,18 +29,22 @@ function Viewer(props) {
   return <span />;
 }
 
+function createHighlightData() {
+  return faker ? [] : [
+    faker.createSpotlightItem(),
+    faker.createSpotlightItem({type: "bookmark"}),
+    faker.createSpotlightItem({isRecommended: true}),
+    faker.createSpotlightItem({override: {syncedFrom: "Nick's iPhone"}}),
+    faker.createSpotlightItem({override: {isOpen: true}})
+  ];
```

Comment:
I'm having trouble understanding this bit.
Isn't it saying "if `faker` is truthy (ie: we're in `__CONFIG__.DEVELOPMENT`), return an empty array, otherwise create some faker spotlight items"? But if `faker` is falsey, wouldn't it fail in the inline `else` clause when we try and call `faker.creteSpotlightItem()`? Or am I missing something obvious.